### ISSUE TITLE
POST /studies lastUpdate field successful but GET /studies lastUpdate is always null

### ISSUE THREAD

**Issue by nickpalladino** [plantbreeding/brapi-Java-TestServer#56]
Example:

POST /studies

```
[
	{
		"active": true,
		"additionalInfo": {},
		"commonCropName": "Blueberry",
		"contacts": [],
		"culturalPractices": null,
		"dataLinks": [],
		"documentationURL": null,
		"endDate": null,
		"environmentParameters": [],
		"experimentalDesign": null,
		"externalReferences": [],
		"growthFacility": null,
		"lastUpdate": {
			"timestamp": "2021-10-07T12:05:18.271944-04:00",
      "version": "testing lastUpdate"
		},
		"license": null,
		"locationDbId": null,
		"locationName": null,
		"observationLevels": [],
		"observationUnitsDescription": null,
		"seasons": [],
		"startDate": null,
		"studyCode": null,
		"studyDescription": "test",
		"studyName": "test study",
		"studyPUI": null,
		"studyType": null,
		"trialDbId": null,
		"trialName": null,
		"studyDbId": null
	}
]
```

GET /studies
```
[
      {
        "active": true,
        "additionalInfo": {},
        "commonCropName": "Blueberry",
        "contacts": [],
        "culturalPractices": null,
        "dataLinks": [],
        "documentationURL": null,
        "endDate": null,
        "environmentParameters": [],
        "experimentalDesign": null,
        "externalReferences": [],
        "growthFacility": null,
        "lastUpdate": null,
        "license": null,
        "locationDbId": null,
        "locationName": null,
        "observationLevels": [],
        "observationUnitsDescription": null,
        "seasons": [],
        "startDate": null,
        "studyCode": null,
        "studyDescription": "test",
        "studyName": "test study",
        "studyPUI": null,
        "studyType": null,
        "trialDbId": null,
        "trialName": null,
        "studyDbId": "1e826807-22ce-41a1-8803-600e15ae8f33"
      }
]
```

**Comment by BrapiCoordinatorSelby** [comment]
fixed in b1c394a03bc7ad12bd4a4ba4ebdbf917b38caae5

**Comment by nickpalladino** [comment]
Thanks!
